Author: Pino Toscano <pino@debian.org>
Description: Fix build with the libraries of linphone >= 3.6.
 - in linphonemediaengine.cc, cast audio_stream_ to the "base" MediaStream
   struct
 - stun.h: make the header guard a bit more specific, so it is different than
   the one of the stun.h provided by libortp
Forwarded: no
Last-Update: 2013-09-08

Index: kopete/protocols/jabber/googletalk/libjingle/talk/session/phone/linphonemediaengine.cc
===================================================================
--- kopete.orig/protocols/jabber/googletalk/libjingle/talk/session/phone/linphonemediaengine.cc	2013-09-07 15:24:31.569025756 +0200
+++ kopete/protocols/jabber/googletalk/libjingle/talk/session/phone/linphonemediaengine.cc	2013-09-09 11:42:21.984318286 +0200
@@ -200,7 +200,7 @@
       LOG(LS_INFO) << "Using " << i->name << "/" << i->clockrate;
       pt_ = i->id;
       audio_stream_ = audio_stream_start(&av_profile, -1, "localhost", port1, i->id, 250, 0); /* -1 means that function will choose some free port */
-      port2 = rtp_session_get_local_port(audio_stream_->session);
+      port2 = rtp_session_get_local_port(((MediaStream *)audio_stream_)->session);
       first = false;
     }
   }
@@ -211,7 +211,7 @@
     // working with a buggy client; let's try PCMU.
     LOG(LS_WARNING) << "Received empty list of codces; using PCMU/8000";
     audio_stream_ = audio_stream_start(&av_profile, -1, "localhost", port1, 0, 250, 0); /* -1 means that function will choose some free port */
-    port2 = rtp_session_get_local_port(audio_stream_->session);
+    port2 = rtp_session_get_local_port(((MediaStream *)audio_stream_)->session);
   }
 
   return true;
